packages/plugin-js-packages/src/lib/runner/outdated/transform.ts

```diff
@@ -18,23 +14,27 @@ export function outdatedResultToAuditOutput(
   const relevantDependencies: OutdatedResult = result.filter(
     dep => dep.type === dependencyGroupToLong[depGroup],
   );
-  // TODO use semver logic to compare versions
-  const outdatedDependencies = relevantDependencies
-    .filter(dep => dep.current !== dep.latest)
-    .filter(
-      dep =>
-        dep.current.split('-')[0]?.toString() !==
-        dep.latest.split('-')[0]?.toString(),
-    );
 
-  const outdatedStats = outdatedDependencies.reduce(
-    (acc, dep) => {
-      const outdatedLevel = getOutdatedLevel(dep.current, dep.latest);
-      return { ...acc, [outdatedLevel]: acc[outdatedLevel] + 1 };
-    },
-    { major: 0, minor: 0, patch: 0 },
+  const validDependencies = relevantDependencies
+    .filter(dep => parse(dep.current) != null && parse(dep.latest) != null)
```

Comment:
Could you comment what you filter out pls

Reply:
I updated the code.